SPARKCONF.md

```diff
@@ -0,0 +1,155 @@
+##Spark Configuration
+
+oni-ml main component uses Spark and Spark SQL to analyze network events and produce a list of least probable events
+or most suspicious. 
+
+Before running oni-ml users need to configure the component. Here are our recommended settings.  
```

Comment:
"Before running oni-ml users need to configure" => "To run oni-ml with its best performance and scalability, it will probably be necessary to configure"